pages/comingSoon/index.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Index from "./index";
+
+describe("ComingSoon index test", () => {
+  it("index page main text", async () => {
+    const { getByText } = render(<Index text="COMING SOON!" />);
+    expect(getByText("COMING SOON!"));
```

Comment:
The right way to do it is something like 
expect(getByText("COMING SOON!")).toBeInTheDocument() // <- This is a helper from jest-dom library you need to add if want to use it
You can do something like :
expect(getByText("COMING SOON!")).toBeDefined();